tests/config/config.js

```diff
@@ -0,0 +1,252 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* globals describe, it, expect, beforeEach */
+
+'use strict';
+
+var modules = bender.amd.require( 'config' );
+
+var config;
+
+beforeEach( function() {
+	var Config = modules.config;
+
+	config = new Config( {
+		creator: 'inline',
+		language: 'pl',
+		resize: {
+			minHeight: 300,
+			maxHeight: 800,
+			icon: {
+				path: 'xyz'
+			}
+		},
+		toolbar: 'top'
+	} );
+} );
+
+describe( 'constructor', function() {
+	it( 'should set configurations', function() {
+		expect( config ).to.have.property( 'creator' ).to.equals( 'inline' );
+		expect( config ).to.have.property( 'language' ).to.equals( 'pl' );
+		expect( config ).to.have.property( 'resize' ).to.have.property( 'minheight' ).to.equals( 300 );
+		expect( config ).to.have.property( 'resize' ).to.have.property( 'maxheight' ).to.equals( 800 );
+		expect( config ).to.have.property( 'resize' ).to.have.property( 'icon' )
+			.to.have.property( 'path' ).to.equals( 'xyz' );
+		expect( config ).to.have.property( 'toolbar' ).to.equals( 'top' );
+	} );
+
+	it( 'should work with no parameters', function() {
+		var Config = modules.config;
```

Comment:
Do you really have to do this in every test case that uses this module? Can't you use `before` hook to make a reference once and reuse it later?